pkg/cloud/ibm/assets/deployment.yaml

```diff
@@ -0,0 +1,132 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ibm-cloud-controller-manager
+  namespace: openshift-cloud-controller-manager
+  labels:
+    k8s-app: ibm-cloud-controller-manager
+spec:
+  replicas: 3
```

Comment:
We use 2 replicas on other clouds, might be good to be consistent 🤔 

(Should probably add another test case for this one)

Reply:
replicas updated